{
	"candidate": "CVE-2016-4806",
	"description": "Web2py versions 2.14.5 and below was affected by Local File Inclusion vulnerability, which allows a malicious intended user to read/access web server sensitive files.",
	"priority": "medium",
	"cvss": {
		"nvd": " CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:N/A:N [7.5 HIGH]"
	},
	"bugs": [
		"http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=856127"
	],
	"references": [
		"https://github.com/web2py/web2py/issues/1585",
		"https://github.com/web2py/web2py/issues/1316",
		"https://github.com/web2py/web2py/commit/1b42fe65472930668435007cfcb077207051ba34",
		"https://www.cve.org/CVERecord?id=CVE-2016-4806"
	],
	"public_date": "2017-01-11 16:59:00 UTC",
	"packages": {
		"bionic": {
			"web2py": {
				"name": "web2py",
				"status": "DNE"
			}
		},
		"devel": {
			"web2py": {
				"name": "web2py",
				"status": "DNE"
			}
		},
		"esm-apps/xenial": {
			"web2py": {
				"name": "web2py",
				"status": "needed"
			}
		},
		"focal": {
			"web2py": {
				"name": "web2py",
				"status": "DNE"
			}
		},
		"groovy": {
			"web2py": {
				"name": "web2py",
				"status": "DNE"
			}
		},
		"hirsute": {
			"web2py": {
				"name": "web2py",
				"status": "DNE"
			}
		},
		"impish": {
			"web2py": {
				"name": "web2py",
				"status": "DNE"
			}
		},
		"jammy": {
			"web2py": {
				"name": "web2py",
				"status": "DNE"
			}
		},
		"kinetic": {
			"web2py": {
				"name": "web2py",
				"status": "DNE"
			}
		},
		"lunar": {
			"web2py": {
				"name": "web2py",
				"status": "DNE"
			}
		},
		"mantic": {
			"web2py": {
				"name": "web2py",
				"status": "DNE"
			}
		},
		"precise/esm": {
			"web2py": {
				"name": "web2py",
				"status": "DNE"
			}
		},
		"trusty": {
			"web2py": {
				"name": "web2py",
				"status": "ignored",
				"note": "end of standard support"
			}
		},
		"trusty/esm": {
			"web2py": {
				"name": "web2py",
				"status": "DNE"
			}
		},
		"upstream": {
			"web2py": {
				"name": "web2py",
				"status": "needs-triage"
			}
		},
		"xenial": {
			"web2py": {
				"name": "web2py",
				"status": "ignored",
				"note": "end of standard support, was needed"
			}
		}
	}
}
